packages/app/src/runner/useRunnerStyle.ts

```diff
@@ -57,12 +59,14 @@ export const useRunnerStyle = ({
   const autStore = useAutStore()
 
   const viewportStyle = computed(() => {
-    let scale: number = 1
+    let scale = 1
```

Comment:
why drop the type number?

Reply:
It's inferred anyway, so I think I just cleaned it up as redundant.